istioctl/pkg/util/configdump/endpoint.go

```diff
@@ -21,9 +21,12 @@ import (
 )
 
 // GetEndpointsConfigDump retrieves the listener config dump from the ConfigDump
-func (w *Wrapper) GetEndpointsConfigDump() (*admin.EndpointsConfigDump, error) {
+func (w *Wrapper) GetEndpointsConfigDump(allowMissingEds bool) (*admin.EndpointsConfigDump, error) {
 	endpointsDumpAny, err := w.getSection(endpoints)
 	if err != nil {
+		if allowMissingEds {
```

Comment:
Do we even need the bool check here?  I feel like if endpoints are not provided we can just always skip? The `includeEds` flag in `setupPodConfigdumpWriter` is always passed in as a hardcoded `true|false` and isn't tied to any user flags.  When would we want to return an error for missing EDS vs returning an empty list?

Reply:
Thanks, It looks like the check here is not necessary, and we can always skip this error since `includeEds` is always used in the requried commands - this means we should have the endpoints for those commands.